my-school/src/Components/AcitvityOverview.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
There is a typo in this filename. Please fix prior to merge

Reply:
just fixed it

Reply:
Thank u sir